### PR TITLE
Convert multi-line strings to the Swift 4 syntax

### PR DESCRIPTION
**Pull request by hartbit**

### CODE REVIEW

**Comment by hartbit** [comment]
@aciidb0mb3r Still a WIP, but I'd appreciate some feedback, especially concerning `ArgumentParserShellCompletion.swift` where I have my doubts

File: Sources/PackageLoading/ModuleMapGenerator.swift [hunk 1]
```diff
@@ -159,12 +159,21 @@ public struct ModuleMapGenerator {
         stream <<< "module \(target.c99name) {\n"
         switch type {
         case .header(let header):
-            stream <<< "    umbrella header \"\(header.asString)\"\n"
```
Comment (aciidb0mb3r) [inline Sources/PackageLoading/ModuleMapGenerator.swift]: I don't really think multiline strings are better in this case

**Comment by aciidb0mb3r** [comment]
Looks good

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Pull request merged**